CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Indigo's target CMake version is 2.8.11 - I'm pretty sure we shouldn't go above this or we might break Ubuntu Saucy (13.10) or something. Can you reduce the version to 11 and have it still work?